Explorer/Assets/Scripts/ECS/StreamableLoading/Textures/LoadTextureSystem.cs

```diff
@@ -34,10 +35,17 @@ protected override async UniTask<StreamableLoadingResult<Texture2DData>> FlowInt
             // Attempts should be always 1 as there is a repeat loop in `LoadSystemBase`
             var result = await webRequestController.GetTextureAsync(
                 intention.CommonArguments,
-                new GetTextureArguments(intention.IsReadable),
+                getTextureArgsFactory.NewArguments(intention.TextureType),
                 GetTextureWebRequest.CreateTexture(intention.WrapMode, intention.FilterMode),
                 ct,
-                GetReportData());
+                GetReportData()
+            );
+
+            if (result == null)
```

Comment:
If you follow my advice and don't try to play with `null` it won't be needed as exceptions are handled on the upper layer automatically and wrapped as required into `StreamableLoadingResult`. Moreover it's a bad practice to create an exception without throwing as it the stack and the context won't be attached

Reply:
Agreed on that it's bad practice to omit context. But the origin contract won't allow to do so, I'd suggest to replace it on Result<T>